docs/data/charts/legend/legend.md

```diff
@@ -16,56 +16,68 @@ In chart components, the legend links series with `label` properties and their c
 
 ## Customization
 
+This section explains how to customize the legend using classes and properties.
+
+In order to fully customize the legend with custom components, see an example at the [HTML components docs](https://mui.com/x/react-charts/components/#html-components).
+
+### Dimensions
+
+Much of the customization can be done using CSS properties.
+There is a main class for the legend container, `.MuiChartsLegend-root`.
+Alternatively the `legendClasses` variable can be used if using CSS in JS to target the elements.
+
+Each legend item is composed of two main elements: the `mark` and the `label`.
+
+The example below explains how it is possible to customize some parts the legend.
+And shows how to use both the `legendClasses` variable and the CSS class directly.
+
+{{"demo": "LegendDimensionNoSnap.js", "hideToolbar": true, "bg": "playground"}}
```

Comment:
Another nitpick, unrelated to this PR: We could probably set a minValue on these inputs here

https://github.com/user-attachments/assets/6f35af3c-7267-485e-8bd3-61bbc3f05672